# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arcticfox/uname-wrapper.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- Arctic-Fox-45.3/media/webrtc/trunk/build/common.gypi.vanilla	2025-04-16 13:32:20.645710810 +0200
+++ Arctic-Fox-45.3/media/webrtc/trunk/build/common.gypi	2025-04-16 13:32:27.835285998 +0200
@@ -117,7 +117,7 @@
             # Anything else gets passed through, which probably won't work very
             # well; such hosts should pass an explicit target_arch to gyp.
             'host_arch%':
-              '<!(uname -m | sed -e "s/i.86/ia32/;s/x86_64/x64/;s/amd64/x64/;s/arm.*/arm/;s/i86pc/ia32/")',
+              '<!(uname -m 2>/dev/null | sed -e "s/i.86/ia32/;s/x86_64/x64/;s/amd64/x64/;s/arm.*/arm/;s/i86pc/ia32/")',
           }],
 
           # Set default value of toolkit_views based on OS.
